src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -636,6 +656,7 @@ void GeneralPage::initLayout(GeneralPageView &layout)
                        s.useCustomFfzModeratorBadges);
     layout.addCheckbox("Use custom FrankerFaceZ VIP badges",
                        s.useCustomFfzVipBadges);
+    layout.addCheckbox("7TV", s.showBadgesSevenTV);
```

Comment:
This setting should be above the split line, directly under FFZ Badge, in my opinion

Reply:
Thanks, I missed that!